uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request-ctrl.js

```diff
@@ -138,7 +138,7 @@ define([
 
         vm.request = TOILRequestInstance.init(attributes);
         //required by leave request so set it to All Day
-        vm.request.to_date_type = vm.request.from_date_type = 1;
+        vm.request.to_date_type = vm.request.from_date_type = '1';
```

Comment:
The comment `//required by leave request so set it to All Day` is not clear, please give more details to the comment.